modules/sdp/SDP.js

```diff
@@ -48,6 +51,39 @@ SDP.prototype.removeTcpCandidates = false;
  */
 SDP.prototype.removeUdpCandidates = false;
 
+/**
+ * Adds a new m-line for receiving source of the given media type.
```

Comment:
Isn't it for sending local tracks? In which case this should say "for sending source" of given media type?

Reply:
Yeah but technically when you add an m-line in the remote description, you are technically adding a receiver for receiving a source, be it local or remote. I can change the wording if its confusing.